streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -229,16 +229,33 @@ public Sensor addLatencyAndThroughputSensor(final String scopeName,
                                                 final String operationName,
                                                 final Sensor.RecordingLevel recordingLevel,
                                                 final String... tags) {
+
+        return addThroughputSensor(null,
```

Comment:
I think you meant `addLatencyAndThroughputSensor`

Reply:
doh! yes, good catch